instrumentation/lettuce/lettuce-4.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/lettuce/v4_0/LettuceSingletons.java

```diff
@@ -16,7 +16,7 @@
 import io.opentelemetry.javaagent.instrumentation.api.instrumenter.PeerServiceAttributesExtractor;
 
 public final class LettuceSingletons {
-  private static final String INSTRUMENTATION_NAME = "io.opentelemetry.javaagent.jedis-1.4";
+  private static final String INSTRUMENTATION_NAME = "io.opentelemetry.lettuce-4.0";
```

Comment:
I guess it was good to go over all those names just to catch copy-paste mistakes like this one 😄